todo_app/data/session_items.py

```diff
@@ -1,20 +1,14 @@
-from todo_app.trello import Trello
-
-global t
-t = Trello()
-
-
-def get_items():
+def get_items(trello):
     """
     Fetches all saved items from the session.
 
     Returns:
         list: The list of saved items.
     """
    
-    return t.get_all_cards()
+    return trello.get_all_cards()
```

Comment:
Next time you're refactoring this file it would be nice to rename it and make it a class with `trello` as a property instead of passing it around, can definitely wait until module 8 though :)